api/authentication_test.go

```diff
@@ -15,7 +16,7 @@ func TestMiddleWareAuthenticationReturnsForbidden(t *testing.T) {
 		So(err, ShouldBeNil)
 		w := httptest.NewRecorder()
 		auth.Check(mockHTTPHandler).ServeHTTP(w, r)
-		So(w.Code, ShouldEqual, http.StatusForbidden)
+		So(w.Code, ShouldEqual, http.StatusNotFound)
```

Comment:
Update the convey message in this test to reflect the change

Reply:
done